stdlib/lang-parser/README.md

```diff
@@ -0,0 +1,309 @@
+## Ballerina-lang-parser
```

Comment:
Shouldn't we have a proper heading here instead?
```suggestion
## Ballerina Parser
```

And then have a small description to say this is a parser written for Ballerina in Ballerina?

Reply:
Ballerina lang-parser implementation in no longer maintained and has already being removed from the ballerina-lang repo.